### PR TITLE
#21189 Pulse Chart shows error messages at the console

### PR DESCRIPTION
**Pull request by vtkalek**

### CODE REVIEW

File: src/visual.ts [hunk 1]
```diff
@@ -1437,7 +1438,8 @@ module powerbi.extensibility.visual {
 
             selection
                 .attr("cx", (d: DataPoint) => xScale(d.categoryValue))
-                .attr("cy", (d: DataPoint) => yScales[d.groupIndex](d.y))
+                .attr("cy", (d: DataPoint) => { return yScales[d.groupIndex](d.y);
```
Comment (ignatvilesov) [inline src/visual.ts]: It'd be better to remove curly brackets and return statement.

File: src/visual.ts [hunk 2]
```diff
@@ -332,6 +332,9 @@ module powerbi.extensibility.visual {
                         description: columns.EventDescription && columns.EventDescription.values && <string>columns.EventDescription.values[categoryIndex],
                     };
                 }
+                let y_value = <number>(y_group0Values && y_group0Values[categoryIndex]) || <number>(y_group1Values && y_group1Values[categoryIndex]) || 0;
+                if (isNaN(y_value))
```
Comment (ignatvilesov) [inline src/visual.ts]: It'd be better to add curly brackets.

File: src/visual.ts [hunk 3]
```diff
@@ -332,6 +332,9 @@ module powerbi.extensibility.visual {
                         description: columns.EventDescription && columns.EventDescription.values && <string>columns.EventDescription.values[categoryIndex],
                     };
                 }
+                let y_value = <number>(y_group0Values && y_group0Values[categoryIndex]) || <number>(y_group1Values && y_group1Values[categoryIndex]) || 0;
```
Comment (ignatvilesov) [inline src/visual.ts]: Do unit tests cover changes that you have made?

Comment (vtkalek) [reply src/visual.ts]: Yes

Comment (uve) [reply src/visual.ts]: I don't see any unit-tests in this PR

Comment (uve) [reply src/visual.ts]: Increase version of the visual and update changelog

Comment (vtkalek) [reply src/visual.ts]: Done

**Pull request merged**